server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -140,7 +172,7 @@ export function create(
 
 	router.delete(
 		"/repos/:ignored?/:tenantId/git/summaries",
-		throttle(throttler, winston, commonThrottleOptions),
+		throttle(restTenantGeneralThrottler, winston, tenantGeneralThrottleOptions),
```

Comment:
@znewton just to confirm, this delete api is internally called by FRS services and not by customers right?